.papr.sh

```diff
@@ -18,6 +18,9 @@ else
 fi
 export target_branch
 
+git fetch origin
```

Comment:
let's just fetch the target branch to save bw.

git fetch origin ${target_branch}